cordova-lib/src/cordova/plugin.js

```diff
@@ -168,10 +168,19 @@ module.exports = function plugin(command, targets, opts) {
                     })
                     .then(function(dir) {
                         // Validate top-level required variables
-                        var pluginVariables = pluginInfoProvider.get(dir).getPreferences(),
-                            missingVariables = pluginVariables.filter(function (v) {
+                        var pluginVariables = pluginInfoProvider.get(dir).getPreferences();
+                        var requiredVariables = [];
+                        for(var i in pluginVariables)
+                        {
+                            var v = pluginVariables[i];
+                            // discard variables with default value
+                            if (!v.default)
```

Comment:
Have a look in `spec-cordova/plugin.spec.js`. You should be able to add something there. Create a `plugin.xml` with the required bits and add it under `spec-cordova/fixtures/plugins`.